blog/_posts/2017-08-10-opensource-here-we-come.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Open source, here we come!
+layout: post
+created_at: Monday August 10 2017
+permalink: blog/2017-08-10-opensource-here-we-come
+current: blog
+author: Gemini
+twitter: Team_Gemini_
+categories:
+- student-posts
+---
+
+# Duo of the moment
+
+![image](/img/blog/2017/TeamGemini.jpg)
+<div align="center" class="image-credits"> Team Gemini(Image Courtesy: Shagun Kapur)</div>
+
+[Akarsha](https://github.com/aksh98) and [Ramya](https://github.com/meriki) are not your regular individuals, a dash of crazy, a drop of weird and a couple of undiscovered unusual ingredients makes them up.  They can be found studying in the same college of IIIT Delhi and sharing the same hostel room, adding a whole new meaning to birds of a feather flock together. Their acquaintance started with their mutual interest in coding and fascination for artificial intelligence. Slowly but steadily started bonding and came to realise that despite one being from North India and the other from South, one being a die hard Korean fan and the other a Game of Thrones fan, they are eerily similar. Their sun signs (which both are extremely fond of) also matched! While looking for the perfect team name, ‘Gemini’ which literally means ‘The Twins’ made perfect sense. In the short span of two years, they have managed to gather all sorts of memories with each other, reading for tests together, doing assignments together, bickering and laughing the next moment, watching youtube videos, heated discussions on random topics or just not seeing eye to eye with one other. They’ve been through it all and vehemently attest that their bond keeps going strong because of all their experiences together.
```

Comment:
Change "Slowly but steadily ..." -> "Slowly but steadily **they** .." , "came to realise" -> "realized"